neptun/src/device/integration_tests/mod.rs

```diff
@@ -477,6 +473,33 @@ mod tests {
         assert!(response.ends_with("errno=0\n\n"));
     }
 
+    #[test]
+    #[ignore]
+    fn test_wireguard_uapi_chaining() {
+        let wg = WGHandle::init("192.0.2.0".parse().unwrap(), "::2".parse().unwrap());
+
+        let response = wg.wg_uapi_cmd("get=1\n\nget=1\n\n");
+        assert_eq!(response.matches("errno=0\n\n").count(), 2);
+
+        let response = wg.wg_uapi_cmd("get=1\n\nset=1\nlisten_port=12345\n\nget=1\n\n");
+        assert_eq!(response.matches("errno=0\n\n").count(), 3);
+    }
+
+    #[test]
+    #[ignore]
+    fn test_wireguard_uapi_set_without_proper_ending() {
```

Comment:
What do you think about some tests that respond with errors?

Reply:
Could add a few, but TBH I don't think there would be much value out of them.